turbinia/turbiniactl.py

```diff
@@ -54,11 +53,223 @@ def csv_list(string):
   return string.split(',')
 
 
-def main():
-  """Main function for turbiniactl"""
-  # TODO(aarontp): Allow for single run mode when
-  # by specifying evidence which will also terminate the task manager after
-  # evidence has been processed.
+def process_evidence(**kwargs):
```

Comment:
Unfortunately this diff is pretty hard to read because github doesn't seem to have any move detection in its diff algorithm.   I suspect that the majority of the diff content is from just adding the new function wrappers.  I know it won't be in alphabetical order, but could we potentially just re-order these two functions (ie. put `process_args()` before `process_evidence()`), and I think (hope) that will produce a more readable diff?  Thanks!

Reply:
You should be able to load diff now